gio/src/file_info.rs

```diff
@@ -47,4 +47,28 @@ impl FileInfo {
             );
         }
     }
+
+    #[doc(alias = "g_file_info_get_attribute_stringv")]
+    #[doc(alias = "get_attribute_stringv")]
+    pub fn attribute_stringv(&self, attribute: &str) -> StrV {
```

Comment:
Why not use IntoGStr here ?

Reply:
I did not do it because I do not think we want to manually impl all attributes getters and setters, and the auto one for now take str

Reply:
But now it's inconsistent :) That's going to be confusing.

Also once the code generator supports these things, `StrV` will also be covered.

Reply:
Oh I misunderstood. Yes I agree, the `attribute` should stay a `&str`. I thought you changed it to `impl IntoGStr` and *that* would then be inconsistent.